### DISCUSSION TITLE
Checkboxes that are bound to array item are sending true instead of the provided value via value="" attribute.

### DISCUSSION THREAD

**Discussion by modrictin** [livewire/livewire#8619] [7. Bugs (archived)]
### Livewire version

3.5.1

### Laravel version

10.48.14

### Which PHP version are you using?

PHP 8.3

### Steps To Reproduce

1. Create a simple livewire component
2. Define a nice array property `$testArray = []`
3. Bind the simple checkbox to the test array but use the key: `wire:model.live="testArray.1"`
4. Define the value of the checkbox as a string: `testValue`
5. See that the sent value sent from Livewire javascript is `true`, instead of `testValue`.
6. Remove the `.1` in the `wire:model` like: `wire:model.live="testArray"`
7. Notice that the value sent is `testValue` now.

Like so:
```PHP
<?php

namespace App\Livewire;

use Livewire\Component;

class Test extends Component
{

    public array $testArray = [];

    public function render()
    {
        return view('livewire.test');
    }
}

```

```HTML
<div>
    <input type="checkbox" value="testValue" wire:model.live="testArray.1">
    @dump($testArray)
</div>
```

### Repository or Wirebox URL

https://wirebox.app/b/4306m

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by joshhanley** [comment]
@modrictin you shouldn't need to specify `.1` on the property name. Just do `wire:model.live="testArray"` then it should work as you'd expect.

Hope this helps!

**Reply by modrictin** [reply]
I’m trying to bind some other data to a special key, dynamically of course. 

The binding itself works perfectly, so it’s not a binding issue. However, the problem lies in the wrong value being sent. 

I was expecting it to work like it did in v2, but it’s not behaving as expected. 😕

Hope this helps! 😊
